test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -675,6 +677,184 @@ public void testGetDocIds() throws Exception {
   }
 
   @Test
+  public void testGetDocIdsActionColumnDelete() throws Exception {
+    executeUpdate("create table data(id integer, action varchar)");
+    executeUpdate("insert into data(id, action) values('1001', 'delete')");
+
+    Map<String, String> configEntries = new HashMap<String, String>();
+    configEntries.put("db.actionColumn", "action");
+    configEntries.put("db.user", "sa");
+    configEntries.put("db.password", "");
+    configEntries.put("db.url", JdbcFixture.URL);
+    configEntries.put("db.uniqueKey", "id:int");
+    configEntries.put("db.everyDocIdSql", "select * from data");
+    configEntries.put("db.singleDocContentSql", "");
+    configEntries.put("db.singleDocContentSqlParameters", "");
+    configEntries.put("db.aclSqlParameters", "id");
+    configEntries.put("adaptor.namespace", "Default");
+    configEntries.put("db.modeOfOperation", "urlAndMetadataLister");
+    configEntries.put("db.modeOfOperation.urlAndMetadataLister.columnName",
```

Comment:
Even in our twisted config world, some of these are weird. This one, for example, doesn't exist and logs a warning if you use it.